### PR TITLE
changed both depth searches, one appears to do nothing

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

**Comment by jasonmotha** [comment]
Fixes a bug where couldn't extract Z2.5 at a certain location. After investigation, we found that there were locations with Z2.5 below 12km which the current code could not extract. 

It looks like one of the extractions is for Velocity Model rather than lat/lon points but unsure (Ethan may be able to clarify).

File: src/callRoutines.c [hunk 1]
```diff
@@ -946,7 +946,7 @@ void runThresholdInputLatLons(char *MODEL_VERSION, char *OUTPUT_DIR, gen_extract
     }
     else if (strcmp(GEN_EXTRACT_VELO_MOD_CALL.VS_TYPE, "Z2.5") == 0)
     {
-        MODEL_EXTENT->Zmax = 12; // 10km downwards
+        MODEL_EXTENT->Zmax = 20; // 10km downwards
```
Comment (jpa198) [inline src/callRoutines.c:949]: 10km at 0.5km increments?

Comment (jasonmotha) [reply src/callRoutines.c:949]: 20km deep at 0.05km increments, so 400 points total now - increased from 240.

Comment (jpa198) [reply src/callRoutines.c:949]: In summary, update your comments :P

Comment (jasonmotha) [reply src/callRoutines.c:949]: TBH, I don't really understand the comments. And going back in time they don't make much sense either... 
image

Comment (jpa198) [reply src/callRoutines.c:949]: image
There should be some method to the madness...

**Review by lee-robin — APPROVED** [review]

File: src/callRoutines.c (hunk 1 above)
Comment (jpa198) [reply src/callRoutines.c:949]: @thomsonem What in tarnation?

**Review by jpa198 — APPROVED** [review]

**Pull request merged**